docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+networks:
+  db-data-net:
+    driver: "bridge"
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: my-sql:1.2
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
```

Comment:
Similarly, the `MYSQL_PASSWORD=1234` is hardcoded. It's recommended to use environment variables or Docker secrets for better security.